pkgs/tools/graphics/lsix/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, stdenv, fetchFromGitHub, makeWrapper, imagemagick }:
+
+stdenv.mkDerivation rec {
+  pname = "lsix";
+  version = "1.7.4";
+
+  src = fetchFromGitHub {
+    owner = "hackerb9";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-mOueSNhf1ywG4k1kRODBaWRjy0L162BAO1HRPaMMbFM=";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    runHook preInstall
+
+    install -d $out/bin
+    install lsix $out/bin
+    wrapProgram $out/bin/lsix --prefix PATH : ${lib.makeBinPath [ imagemagick ] }
```

Comment:
```suggestion
    install -Dm755 lsix -t $out/bin
```

Reply:
Thanks! I didn't know `install` in coreutils has this `-t` flag.